tools/ratbagctl.in

```diff
@@ -495,6 +496,19 @@ def func_profile_get(r, args):
     print_profile(d, p, 0)
 
 
+def func_profile_name_get(r, args):
+    p, d = find_profile(r, args)
+    print(p.name)
+
+
+def func_profile_name_set(r, args):
+    p, d = find_profile(r, args)
+    if RatbagdProfile.CAP_WRITABLE_NAME not in p.capabilities:
+        raise RatbagErrorCapability("assigning a profile name is not supported on this profile")
```

Comment:
You never catch this exception, is that intentional?

Reply:
yep, the exception is forwarded to the caller (ratbagctl) and that's a way to notify the user that he made a mistake.